manifests/supervisorcluster/1.25/cns-csi.yaml

```diff
@@ -90,6 +90,9 @@ rules:
   - apiGroups: [ "snapshot.storage.k8s.io" ]
     resources: [ "volumesnapshotcontents/status" ]
     verbs: [ "update", "patch" ]
+  - apiGroups: ["cns.vmware.com"]
```

Comment:
Sometimes the k8s yaml in CSI repo can be one version ahead, because WCP release can skip the latest k8s yamls due to some regression close to the release. So, n-4 is a safer option for k8s yaml updates